src/app/rundown-view/services/action-trigger-producer-key-binding.service.ts

```diff
@@ -89,14 +91,47 @@ export class ActionTriggerProducerKeyBindingService implements KeyBindingService
     this.actionTriggersWithAction.clear()
 
     this.actionTriggers.forEach(actionTrigger => {
-      const actionForTrigger: Tv2Action | undefined = this.actions.find(action => action.id === actionTrigger.actionId)
-      if (!actionForTrigger) {
+      let action: Tv2Action | undefined = this.actions.find(action => action.id === actionTrigger.actionId)
+      if (!action) {
         return
       }
-      this.actionTriggersWithAction.set(actionTrigger, actionForTrigger)
+      if (action.type === PlaceholderActionType.CONTENT) {
```

Comment:
Would it be an issue to instead say: 
```
if (action?.type === PlaceholderActionType.CONTENT) 
````
Then we can get rid of the first action check.

Reply:
I guees not. It's gone with some of the changes from Anders suggestions.